chia/cmds/chia.py

```diff
@@ -41,7 +41,7 @@ def monkey_patch_click() -> None:
 
     import click.core
 
-    click.core._verify_python3_env = lambda *args, **kwargs: 0  # type: ignore[attr-defined]
+    click.core._verify_python3_env = lambda *args, **kwargs: 0
```

Comment:
Don't remove this I think.

```suggestion
    click.core._verify_python3_env = lambda *args, **kwargs: 0  # type: ignore[attr-defined]
```

Reply:
it was complaining about unused ignore mark

Reply:
maybe related to mypy issue with CI I was seeing